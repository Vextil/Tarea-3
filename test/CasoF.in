CrearLista 1 mueslera pereira gimenez godin  FIN
InsertarCP sanchez 19 arevalo 17 lodeiro 14 rodriguez 7 rolan 19 cavani 21 mayada 19 FIN
ExtraerCP
ElementosCP 19 2
ImprimirLista 2
ConstruirSet 1 1
ConstruirSet 2 2
UnionInterDif 1 2 3 4 5 6
PerteneceASet 3 1
PerteneceASet 4 1
PerteneceASet 5 1
PerteneceASet 6 1
IngresarSolicitudes notepad 1 tomcat 1 jetty 2 gvsig 2 ionic 3 geoserver 1 qgis 3 gvsig 5 firefox 4 chrome 4 gimp 6 FIN
EnAlgunoSolicitudes 1 3 7
EnAmbosSolicitudes 2 5 8
SoloEnUnoSolicitudes 2 5 9
CrearLista 3 gvsig notepad qgis jetty  FIN
PerteneceASet 8 3
PerteneceASet 7 3
PerteneceASet 9 3
CorrerProcesos notepad 60 tomcat 60 jetty 20 ionic 30 geoserver 40 FIN 
PedirRecursos notepad 1 tomcat 1 jetty 3 ionic 3 jetty 4 geoserver 3 notepad 2 notepad 80 notepad 1 tomcat 1 geoserver 3 FIN
SolicitanOMuyConsumidores 30 1
SolicitanOMuyConsumidores 101 4
SolicitanOMuyConsumidores 1 1
Fin
