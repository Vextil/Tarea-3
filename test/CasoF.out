1>OK
2>OK
3>rodriguez - 7
4>OK
5>NO ordenada. sanchez rolan mayada 
6>OK
7>OK
8>OK
9>SI SI SI SI 
10>NO NO NO NO 
11>SI SI SI SI 
12>NO NO NO NO 
13>OK
14>OK
15>OK
16>OK
17>OK
18>SI NO NO NO 
19>NO SI SI NO 
20>NO NO NO SI 
21>OK
22>OK
23>boot geoserver notepad tomcat 
24>jetty 
25>boot geoserver ionic jetty notepad tomcat 
26>Fin
